documentation/docs/20-core-concepts/40-page-options.md

```diff
@@ -142,6 +142,16 @@ Disabling CSR does not ship any JavaScript to the client. This means:
 * `<script>` tags inside all Svelte components are removed.
 * `<form>` elements cannot be [progressively enhanced](form-actions#progressive-enhancement).
 * Links are handled by the browser with a full-page navigation.
+* Hot Module Replacement (HMR) will be disabled.
+
+Setting the value of `csr` to [dev](modules#$app-environment-dev) will enable HMR during development.
```

Comment:
we could be a tiny bit more explicit that we're not _just_ enabling HMR, we're enabling everything else that comes with CSR; HMR is just one of the more obvious reasons you might want to do that. otherwise we're kinda on the hook for people not realising they broke their progressively enhanced forms etc...

```suggestion
You can enable `csr` during development (for example to take advantage of HMR) like so:
```

Reply:
Done, thank you for the review :)